manifests/cluster-authentication-operator_02_config.crd.yaml

```diff
@@ -3,12 +3,109 @@ kind: CustomResourceDefinition
+                - 'Force'
+                - 'Managed'
+                - 'Unmanaged'
+                - 'Removed'
+            observedConfig:
+              description: observedConfig holds a sparse config that controller has
+                observed from the cluster state.  It exists in spec because it is
+                an input to the level for the operator
+              type: object
+            operandSpecs:
+              description: operandSpecs provide customization for functional units
+                within the component
+              items:
+                properties:
+                  name:
+                    description: name is the name of this unit.  The operator must
+                      be aware of it.
+                    type: string
+                  operandContainerSpecs:
+                    description: operandContainerSpecs are per-container options
+                    items:
+                      properties:
+                        name:
+                          description: name is the name of the container to modify
+                          type: string
+                        resources:
+                          description: resources are the requests and limits to place
+                            in the container.  Nil means to accept the defaults.
+                          type: object
+                      type: object
+                    type: array
+                  unsupportedResourcePatches:
+                    description: 'unsupportedResourcePatches are applied to the workload
+                      resource for this unit. This is an unsupported workaround if
+                      anything needs to be modified on the workload that is not otherwise
+                      configurable. TODO Decide: alternatively, we could simply include
+                      a RawExtension which is used in place of the "normal" default
+                      manifest'
+                    items:
+                      properties:
+                        patch:
+                          description: patch the patch itself
+                          type: string
+                        type:
+                          description: 'type is the type of patch to apply: jsonmerge,
+                            strategicmerge'
+                          type: string
+                      type: object
+                    type: array
+                type: object
+              type: array
+            unsupportedConfigOverrides:
+              description: 'unsupportedConfigOverrides holds a sparse config that
+                will override any previously set options.  It only needs to be the
+                fields to override it will end up overlaying in the following order:
+                1. hardcoded defaults 2. observedConfig 3. unsupportedConfigOverrides'
+              type: object
+          type: object
+        status:
+          type: object
+  version: v1
+status:
```

Comment:
Should be unset

Reply:
fixed, and now the only additions are the validation section.